target/scripts/startup/check-stack.sh

```diff
@@ -60,3 +60,11 @@ function _check_log_level
     LOG_LEVEL="${DEFAULT_LOG_LEVEL}"
   fi
 }
+
+function _check_fetch_get_mail
+{
+  if [[ ${ENABLE_FETCHMAIL} -eq 1 ]] && [[ ${ENABLE_GETMAIL} -eq 1 ]]
+  then
+    _shutdown 'Enabling Fetchmail and Getmail at the same time is unsupported'
```

Comment:
What's the reason for that? Why can't someone use both at the same time?

Reply:
Good question, I just kinda assumed people would use one or the other, I don't think there is an actual conflict that would stop it though.

Reply:
I've removed the exclusivity check.